geometry/symmetric_bilinear_form_body.hpp

```diff
@@ -111,6 +111,16 @@ SymmetricBilinearForm<Scalar, Frame, Multivector>::Anticommutator() const {
       matrix_.Trace() * R3x3Matrix<double>::Identity() - matrix_);
 }
 
+template<typename Scalar,
+         typename Frame,
+         template<typename, typename> typename Multivector>
```

Comment:
There are places in this file and others where you give names to the first two typenames thus: `template<typename S, typename F> typename Multivector`.  Please change everything to be consistent.

I would personally favour dropping the `S` and `F` everywhere, but it's your call.